meson.build

```diff
@@ -1,5 +1,6 @@
 project('wrapdb',
   meson_version: '>=0.55.0'
+  default_options: ['cpp_std=c++17'],
```

Comment:
This broke meson.build, because of missing comma at the end of `meson_version` line.